src/test/java/pages/constructor/ConstructorPage.java

```diff
@@ -12,8 +13,9 @@ public class ConstructorPage {
     private final By TOPPING_BUN = By.xpath("//p[contains(text(), 'Краторная булка N-200i')]");
     private final By SOURCE_ELEMENT = By.xpath("//*[@id=\"root\"]/div/main/div/section[2]/div[1]");
     private final By ORDER = By.xpath("//button[contains(text(), 'Оформить заказ')]");
-    private final By ORDER_NUMBER = By.xpath("//*[contains(@class, 'modal-overlay_overlay__AKixE')]");
-    private final By ORDER_FEED = By.xpath("//button[contains(text(), 'Лента заказов')]");
+    private final By ORDER_FEED = By.xpath("//*[@id=\"root\"]/div/header/nav/div[1]/a[2]/p");
+    private final By FIND_ORDER = By.xpath("//*[@id=\"modals\"]/div[1]/div[2]/h2");
+    private final By FIND_ORDER_ON_ORDER_FEED = By.xpath("//*[@id=\"root\"]/div/main/div/div[1]/div/a[3]/div[1]/span[1]");
 
     public SelenideElement toppingBun() {
         return element(TOPPING_BUN).shouldBe(visible);
```

Comment:
Везде где используешь visible добавляй because, так будет легче понять в чем ошибка при падении теста
return element(TOPPING_BUN).shouldBe(visible.because("Сообщение, которое выведется при падении"))

Reply:
исправил